qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -126,9 +126,12 @@ def __init__(
             Optional[Callable] url_resolver: Function used to resolve the runtime url.
             Optional[str] region: Set a region preference. Accepted values are ``us-east`` or ``eu-de``.
                 An instance with this region will be prioritized if an instance is not passed in.
-            Optional[List[str]] plans_preference: A list of account types, ordered by preference.
-                An instance with the first value in the list will be prioritized if an instance
-                is not passed in.
+            Optional[List[str]] plans_preference: A list of account plan names
+                (``open``, ``premium``, etc.), ordered by preference. An instance with the first
+                value in the list will be prioritized and only instances
+                with the given plan names will be available. For example, if you want to avoid
+                using your premium accounts you can just pass in "open" to only use your open plan
+                instances. ``plans_preference`` is ignored if the instance is specified.
```

Comment:
```suggestion
                with the given plan names will be considered. For example, if you want to avoid
                using your premium accounts you can just pass in "open" to only use your open plan
                instances. ``plans_preference`` is ignored if an instance is specified.
```